dashboard/app.yaml

```diff
@@ -33,7 +33,7 @@ resources:
 env_variables:
   REDISHOST: '10.25.27.107'
   REDISPORT: '6379'
-  TEST_NAME_PREFIXES: 'pt-nightly,pt-2.0,tf.nightly,tf.r2.12.0,tf.r2.11.1,%-1vm,jax,flax,pax,mp-jax,mp-pax'
```

Comment:
Thanks for cleaning up the old tests. Can you separate this into a new PR to keep this one focused on the nightly tests?